src/core/config.py

```diff
@@ -1,15 +1,14 @@
 import os
-import google.generativeai as Client
 from github import Github
-from typing import Tuple
 
 class Config:
     GITHUB_TOKEN = os.environ["GITHUB_TOKEN"]
```

Comment:
Similarly, this line imports `typing.Tuple`, which also appears unused.  Please remove it.